### PR TITLE
FAQ: What are packets

### PR DESCRIPTION
**Pull request by Himura2la**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 Frequently asked questions and possible issues you may face when working with [Cloud4RPi](https://cloud4rpi.io):
 
+## What are packets?
+
+Packet is a data portion sent from a device to the Cloud4RPi server (via the [MQTT](/api/mqtt/#cloud4rpi-mqtt-broker) or [HTTPs](/api/http/) protocol). A single packet can include multiple variable values. [Diagnostic data](/api/mqtt/#diagnostic-data) transfer is unlimited and does not consume packets.
```
Comment (arminal) [inline docs/faq.md]:
```suggestion
Packet is a data portion sent from a device to the Cloud4RPi server (using the [MQTT](/api/mqtt/#cloud4rpi-mqtt-broker) or [HTTPs](/api/http/) protocol). A single packet can include multiple variable values. [Diagnostic data](/api/mqtt/#diagnostic-data) transfer is unlimited and does not consume packets.
```

**Review by sky3d — APPROVED** [review]

File: docs/faq.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 Frequently asked questions and possible issues you may face when working with [Cloud4RPi](https://cloud4rpi.io):
 
+## What are packets?
+
+Packet is a data portion sent from a device to the Cloud4RPi server (using the [MQTT](/api/mqtt/#cloud4rpi-mqtt-broker) or [HTTPs](/api/http/) protocol). A single packet can include multiple variable values. [Diagnostic data](/api/mqtt/#diagnostic-data) transfer is unlimited and does not consume packets.
+
+Packets information is displayed at the top right corner of Cloud4RPi pages and on the [Account](https://stage.cloud4rpi.io/account) page.
```
Comment (arminal) [inline docs/faq.md]:
```suggestion
Packet information is displayed at the top right corner of Cloud4RPi pages and on the [Account](https://stage.cloud4rpi.io/account) page.
```

File: docs/faq.md [hunk 3]
```diff
@@ -85,7 +96,7 @@ sudo systemctl daemon-reload
 
 ## How to choose the DATA_SENDING_INTERVAL value?
 
-You should call [device.publish_data()](/api/python/#publish_data) and [device.publish_diag()](/api/python/#publish_diag) functions frequently to update the monitored variables. The update frequency depends on the variable dynamics, for example, atmospheric conditions do not change every minute. This means the delay between updates for a weather station should be one to ten minutes.
+You should call [device.publish_data()](/api/python/#publish_data) and [device.publish_diag()](/api/python/#publish_diag) functions frequently to update the monitored variables. The update frequency depends on the variable dynamics, for example, atmospheric conditions do not change every minute. This means the delay between updates for a weather station should be one to ten minutes. You should also consider available [packets](#what-are-packets) when choosing the `DATA_SENDING_INTERVAL` value.
```
Comment (arminal) [inline docs/faq.md]:
```suggestion
Call [device.publish_data()](/api/python/#publish_data) and [device.publish_diag()](/api/python/#publish_diag) functions frequently to update the monitored variables. The update frequency depends on the variable dynamics, for example, atmospheric conditions do not change every minute. This means the delay between updates for a weather station should be one to ten minutes. You should also consider available [packets](#what-are-packets) when choosing the `DATA_SENDING_INTERVAL` value.
```

**Comment by Himura2la** [comment]
@sky3d Ready to merge

**Pull request merged**